File: src/main/java/org/example/gogoma/controller/response/UserMarathonDetailResponse.java
package org.example.gogoma.controller.response;

import lombok.Builder;
import lombok.Getter;
import org.example.gogoma.domain.usermarathon.dto.UserMarathonDetailDto;
import org.example.gogoma.domain.usermarathon.dto.UserMarathonSearchDto;

import java.util.List;

@Getter
@Builder
public class UserMarathonDetailResponse {

    private UserMarathonDetailDto userMarathonDetailDto;

    public static UserMarathonDetailResponse of(UserMarathonDetailDto userMarathonDetailDto) {
        return UserMarathonDetailResponse.builder()
                .userMarathonDetailDto(userMarathonDetailDto)
                .build();
    }
}

File: app/src/main/java/com/example/gogoma/data/model/UserMarathonDetailResponse.kt
package com.example.gogoma.data.model

import android.os.Parcelable
import com.example.gogoma.data.dto.UserMarathonDetailDto
import kotlinx.parcelize.Parcelize

@Parcelize
data class UserMarathonDetailResponse (
    var userMarathonDetail: UserMarathonDetailDto
) : Parcelable